config/defaults.js

```diff
@@ -69,7 +69,10 @@ module.exports = {
     // Number of times to retry requests to the task queue
     maxRetries: 5,
     // Amount of time to wait between retries
-    requestRetryInterval: 2 * 1000
+    requestRetryInterval: 2 * 1000,
+    // Azure currently has a max of 32 messages that can be retrieved from the
+    // queue at a time.
+    maxMessagesPerRequest: 32
```

Comment:
Funny you mention that, I originally had it as a const but then was unsure if this should be configurable.  Thought that perhaps there might be a time where we might want to decrease/increase this, but was unsure.